src/objects/Sprite.js

```diff
@@ -93,6 +98,22 @@ Sprite.prototype = Object.assign( Object.create( Object3D.prototype ), {
 
 		}
 
+		var barycoord = new Vector3();
+
+		function uvIntersection( point, p1, p2, p3, uv1, uv2, uv3 ) {
+
+			Triangle.getBarycoord( point, p1, p2, p3, barycoord );
+
+			uv1.multiplyScalar( barycoord.x );
+			uv2.multiplyScalar( barycoord.y );
+			uv3.multiplyScalar( barycoord.z );
+
+			uv1.add( uv2 ).add( uv3 );
+
+			return uv1.clone();
```

Comment:
```javascript
return result;
```

Reply:
https://github.com/mrdoob/three.js/blob/dev/src/objects/Mesh.js#L143-L160

Is the code here should also be modified?